upload/admin/view/template/extension/module/smaily_for_opencart.tpl

```diff
@@ -392,20 +392,69 @@
+                    <tr>
+                      <th scope="col"><?php echo $cart_status_table_header_id; ?></th>
+                      <th class="text-left"><a href="<?php echo $cart_status_table_sort_name_link; ?>" class="<?php echo $sort ==='lastname' ? 'strtolower($order)' : '' ?>"><?php echo $cart_status_table_header_name; ?></a></th>
+                      <th class="text-left"><a href="<?php echo $cart_status_table_sort_email_link; ?>" class="<?php echo $sort ==='email' ? 'strtolower($order)' : '' ?>"><?php echo $cart_status_table_header_email; ?></a></th>
+                      <th class="text-left"><?php echo $cart_status_table_header_cart; ?></th>
+                      <th class="text-left"><a href="<?php echo $cart_status_table_sort_date_link; ?>" class="<?php echo $sort ==='sent_time' ? 'strtolower($order)' : '' ?>"><?php echo $cart_status_table_header_date; ?></a></th>
+                      <th class="text-left"><a href="<?php echo $cart_status_table_sort_status_link; ?>" class="<?php echo $sort ==='is_sent' ? 'strtolower($order)' : '' ?>"><?php echo $cart_status_table_header_status; ?></a></th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <?php foreach($abandoned_cart_list as $abandoned_cart): ?>
+                    <tr>
+                      <th scope="row"><?php echo $abandoned_cart['customer_id']; ?></th>
+                        <td><?php echo $abandoned_cart['firstname'] . $abandoned_cart['lastname']; ?></td>
+                        <td><?php echo $abandoned_cart['email']; ?></td>
+                      <td>
+                        <table class="table table-bordered" >
+                          <tbody >
+                            <?php foreach($abandoned_cart['products'] as $product) : ?>
+                            <tr>
+                              <td width="70%"><a href="<?php echo $product_url_without_id . $product['data']['product_id']; ?>"><?php echo $product['data']['name']; ?></a></td>
+                              <td width="15%"><?php echo "x " . $product['quantity']; ?></td>
+                              <td width="15%"><?php echo "$" . $product['data']['price']; ?></td>
+                            </tr>
+                            <?php endforeach; ?>
+                          </tbody>
+                        </table>
+                      </td>
+                      <td><?php echo $abandoned_cart['sent_time'] ?: ''; ?><br></td>
+                      <td><b><?php echo $abandoned_cart['is_sent'] == '1' ? 'SENT' : 'PENDING'; ?><b></td>
+                    </tr>
+                    <?php endforeach; ?>
+                  </tbody>
+                </table>
+                <div class="row">
+                  <div class="col-sm-6 text-left"><?php echo $pagination; ?></div>
+                  <div class="col-sm-6 text-right"><?php echo $results; ?></div>
+                </div>
+              </div>
+            </div>
```

Comment:
Formatting: indentation.

Reply:
I don't see it right away. You mean this?
```
                       <td scope="row"><?php echo $abandoned_cart['customer_id']; ?></td>
-                        <td><?php echo $abandoned_cart['firstname'] . " " . $abandoned_cart['lastname']; ?></td>
-                        <td><?php echo $abandoned_cart['email']; ?></td>
+                      <td><?php echo $abandoned_cart['firstname'] . " " . $abandoned_cart['lastname']; ?></td>
+                      <td><?php echo $abandoned_cart['email']; ?></td>
```

Reply:
I guess so. Now it seems to be fixed.